src/Frontend/state/actions/resource-actions/__tests__/preference-actions.test.ts

```diff
@@ -155,4 +156,53 @@ describe('getOriginIdsToPreferOver', () => {
     const expectedOriginUuids = ['originUuid1'];
     expect(actualOriginUuids).toEqual(expectedOriginUuids);
   });
+
+  it('finds all origin ids for attribution with multiple root resources', () => {
+    const testSource = faker.opossum.source();
```

Comment:
we don't gain anything by prefixing variables in test with "test" or "mock".
```suggestion
    const testSource = faker.opossum.source();
```

Reply:
true, was a left over of the old test naming conventions...